website/docs/src/components/SideBar/SideBarContent.js

```diff
@@ -0,0 +1,380 @@
+  font-weight: 600;
+  padding: ${magma.spaceScale.spacing04} ${magma.spaceScale.spacing05};
+`;
+
+const SubNavWrapper = styled.ul`
+  padding: 0;
+  margin: 0;
+  a {
+    padding: 6px 24px;
+    &:active {
+      background: ${magma.colors.neutral06};
+      position: relative;
+    }
+    &:active:before {
+      content: '';
+      position: absolute;
+      width: 4px;
+      height: 100%;
+      left: 0;
+      top: 0;
+      border-radius: 2px;
+      background: ${magma.colors.primary};
+    }
+  }
+`;
+
+const StyledAccordionPanel = styled(AccordionPanel)`
+  background: none;
+  font-size: ${magma.typeScale.size02.fontSize};
+  padding: 0;
+`;
+
+const StyledAccordionSubHeader = styled.label`
+  font-size: ${magma.typeScale.size01.fontSize};
+  font-weight: 700;
+  text-transform: uppercase;
+  display: inline-block;
+  text-indent: 0;
+  padding-left: 16px;
+  padding-top: 20px;
+`;
+
+export const SideBarContent = (
+  <>
+    <StyledAccordion>
+      <AccordionItem>
+        <h3>
+          <StyledAccordionButton>Designing</StyledAccordionButton>
+        </h3>
+        <SubNavWrapper>
+          <StyledAccordionPanel>
+            <Hyperlink>Getting started</Hyperlink>
+          </StyledAccordionPanel>
+          <StyledAccordionPanel>
+            <Hyperlink>Color</Hyperlink>
+          </StyledAccordionPanel>
+          <StyledAccordionPanel>
+            <Hyperlink>Layout</Hyperlink>
+          </StyledAccordionPanel>
+          <StyledAccordionPanel>
+            <Hyperlink>Spacing</Hyperlink>
+          </StyledAccordionPanel>
+          <StyledAccordionPanel>
+            <Hyperlink>Typography</Hyperlink>
+          </StyledAccordionPanel>
+        </SubNavWrapper>
+      </AccordionItem>
+
+      <AccordionItem>
+        <h3>
+          <StyledAccordionButton>Developing</StyledAccordionButton>
+        </h3>
+        <SubNavWrapper>
+          <StyledAccordionPanel>
+            <Hyperlink>Getting started</Hyperlink>
+          </StyledAccordionPanel>
+          <StyledAccordionPanel>
+            <Hyperlink>Usage</Hyperlink>
+          </StyledAccordionPanel>
+          <StyledAccordionPanel>
+            <Hyperlink>Custom styles</Hyperlink>
+          </StyledAccordionPanel>
+          <StyledAccordionPanel>
+            <Hyperlink>Versions</Hyperlink>
+          </StyledAccordionPanel>
+          <StyledAccordionPanel>
+            <Hyperlink>Changelog</Hyperlink>
+          </StyledAccordionPanel>
+        </SubNavWrapper>
+      </AccordionItem>
+
+      <AccordionItem>
+        <h3>
+          <StyledAccordionLink>
+            <Hyperlink>Contributing</Hyperlink>
+          </StyledAccordionLink>
+        </h3>
+      </AccordionItem>
+
+      <AccordionItem>
+        <h3>
+          <StyledAccordionButton>Components</StyledAccordionButton>
+        </h3>
+        <StyledAccordionPanel>
+          <h3>
+            <StyledAccordionSubHeader>Layout</StyledAccordionSubHeader>
+          </h3>
```

Comment:
This isn't working as intended because it requires a change in the Gatsby config. We can do this later though, but for now, I wonder if we should have this here structure this way if it's not functional.

Reply:
Old sidebar file.